File: offsets.py
'''
Created using https://github.com/a2x/cs2-dumper
Wed, 28 Feb 2024 22:49:49 +0000
'''

class client_dll: # client.dll
    dwEntityList = 0x18BBAF8
    dwForceAttack = 0x1728F90
    dwForceAttack2 = 0x1729020
    dwForceBackward = 0x1729260
    dwForceCrouch = 0x1729530
    dwForceForward = 0x17291D0
    dwForceJump = 0x17294A0
    dwForceLeft = 0x17292F0
    dwForceRight = 0x1729380
    dwGameEntitySystem = 0x19D9450
    dwGameEntitySystem_getHighestEntityIndex = 0x1510
    dwGameRules = 0x1918A30
    dwGlobalVars = 0x1723B10
    dwGlowManager = 0x1918E50
    dwInterfaceLinkList = 0x1A0A598
    dwLocalPlayerController = 0x190B308
    dwLocalPlayerPawn = 0x1730118
    dwPlantedC4 = 0x1921858
    dwPrediction = 0x172FFE0
    dwSensitivity = 0x1919778
    dwSensitivity_sensitivity = 0x40
    dwViewMatrix = 0x191CF30
    dwViewRender = 0x191D7B0

class engine2_dll: # engine2.dll
    dwBuildNumber = 0x513574
    dwNetworkGameClient = 0x512AC8
    dwNetworkGameClient_getLocalPlayer = 0xF0
    dwNetworkGameClient_maxClients = 0x250
    dwNetworkGameClient_signOnState = 0x240
    dwWindowHeight = 0x5CBC64
    dwWindowWidth = 0x5CBC60

class game_info: # Some additional information about the game at dump time
    buildNumber = 0x36AB # Game build number

class inputsystem_dll: # inputsystem.dll
    dwInputSystem = 0x367A0

File: offsets.rs
// Offsets are automatically updated by frezzyhook.com.
// 2024-05-08 04:40:40.591862300 UTC

#![allow(non_upper_case_globals, non_camel_case_types, unused)]

pub mod cs2_dumper {
    pub mod offsets {
        // Module: client.dll
        pub mod client_dll {
            pub const dwCSGOInput: usize = 0x1930C30;
            pub const dwEntityList: usize = 0x18C7F88;
            pub const dwGameEntitySystem: usize = 0x19E56E0;
            pub const dwGameEntitySystem_getHighestEntityIndex: usize = 0x1510;
            pub const dwGameRules: usize = 0x1924EE0;
            pub const dwGlobalVars: usize = 0x172EEE0;
            pub const dwGlowManager: usize = 0x1925300;
            pub const dwLocalPlayerController: usize = 0x1917698;
            pub const dwLocalPlayerPawn: usize = 0x173B568;
            pub const dwPlantedC4: usize = 0x192E0D8;
            pub const dwPrediction: usize = 0x173B430;
            pub const dwSensitivity: usize = 0x1925C28;
            pub const dwSensitivity_sensitivity: usize = 0x40;
            pub const dwViewAngles: usize = 0x1935FC0;
            pub const dwViewMatrix: usize = 0x1929420;
            pub const dwViewRender: usize = 0x1929C20;
        }
        // Module: engine2.dll
        pub mod engine2_dll {
            pub const dwNetworkGameClient: usize = 0x515AD8;
            pub const dwNetworkGameClient_deltaTick: usize = 0x258;
            pub const dwNetworkGameClient_getLocalPlayer: usize = 0xF0;
            pub const dwNetworkGameClient_getMaxClients: usize = 0x250;
            pub const dwNetworkGameClient_signOnState: usize = 0x240;
            pub const dwWindowHeight: usize = 0x5CF1FC;
            pub const dwWindowWidth: usize = 0x5CF1F8;
        }
        // Module: inputsystem.dll
        pub mod inputsystem_dll {
            pub const dwInputSystem: usize = 0x377C0;
        }
        // Module: matchmaking.dll
        pub mod matchmaking_dll {
            pub const dwGameTypes: usize = 0x1D2290;
            pub const dwGameTypes_mapName: usize = 0x1D23B0;
        }
    }
}
